README.md

```diff
@@ -4,19 +4,13 @@
 
 A generator for [Yeoman](http://yeoman.io) that scaffolds a [Bespoke.js](http://markdalgleish.com/projects/bespoke.js) presentation.
 
-The boilerplate project includes a [Gulp](http://gulpjs.com) build system, a preview server with [LiveReload](http://livereload.com), [Jade](http://jade-lang.com) and [Stylus](http://learnboost.github.io/stylus) compilation, a [GitHub Pages](http://pages.github.com) deployment task, and .
-
-Optionally, PDFs of your presentation can be generated using [bespoke-pdf](https://github.com/davidmarkclements/bespoke-pdf).
-This will mean a 100mb+ download (for nw.js) but it's cached once for all bespoke projects.
+The boilerplate project includes a [Gulp](http://gulpjs.com) build system, a preview server with [LiveReload](http://livereload.com), [Pug (formerly Jade)](http://jade-lang.com) and [Stylus](http://stylus-lang.com/) compilation, a [GitHub Pages](http://pages.github.com) deployment task, and .
```

Comment:
This sentence is incomplete. It is also incomplete in master, so it's probably just a carry over. We should either end it with "and a GitHub Pages deployment task" or add additional points to finish off the sentence.